tests/pytests/test_comprehensive_jk_screening.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+import psi4
+import itertools
+from utils import compare, compare_integers, compare_values
+
+pytestmark = [pytest.mark.psi, pytest.mark.api] 
+
+@pytest.mark.parametrize("scf_type", [ "PK", "DIRECT", "OUT_OF_CORE", "DISK_DF", "MEM_DF", "DFDIRJ+LINK", "DFDIRJ+COSX" ])
+@pytest.mark.parametrize("scf_subtype", [ "AUTO", "INCORE", "OUT_OF_CORE", "YOSHIMINE_OUT_OF_CORE", "REORDER_OUT_OF_CORE" ])
+@pytest.mark.parametrize("screening", [ "SCHWARZ", "DENSITY", "CSAM", "NONE" ])
+def test_comprehensive_jk_screening(scf_type, scf_subtype, screening):
+    """Checks the energy values computed by different JK methods using different
+    screening types. The differences in energies should be insignificant.""" 
+
+    #== define reference energies ==#
+    Eref = {  
+        "Nuclear"       :   30.7884922572,     #TEST
+        "Singlet": {
+            "Canonical" : -149.58723684929720, #TEST
+            "DF"        : -149.58715054487624, #TEST
+            "Composite": {
+              "DFDIRJ+COSX"    : -149.58722317236171, #TEST
+              "DFDIRJ+LINK"    : -149.58726772171027  #TEST
```

Comment:
fwiw, the `#TEST` don't do anything in `tests/pytests/`

Reply:
Ah, good to know. I will probably remove it, then.